src/executables/util_service.c

```diff
@@ -1442,6 +1396,61 @@ process_service (int command_type, bool process_window_service)
   return status;
 }
 
+/*
+ * get_server_names -
+ *
+ * return:
+ *
+ *      out_buffer (out):
+ */
+static void
+get_server_names (char *out_buffer)
```

Comment:
I think that this function collect server name which execute by heartbeat and return collecting value via out_buffer.
So, how about change function name from get_server_names() to get_ha_servers() ?

Reply:
It collects the name of the non-HA servers.

Reply:
sorry, since I am not good at C++ function, I mis-understand.